src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs

```diff
@@ -1,337 +1,339 @@
-                    .Pay.Content.Should.BeEquivalent($"Betala {string.Format("{0:N2}", Convert.ToDecimal(products.Sum(x => x.UnitPrice / 100 * x.Quantity)))} kr")
-                    .Pay.ClickAndGo();
-
-                case Checkout.Option.LocalPaymentMenu:
-                case Checkout.Option.Anonymous:
-                default:
-
-                    return GoToPayexSwishPaymentFrame(products, checkout)
-                    .SwishNumber.Set(info.SwishNumber)
-                    .Pay.Content.Should.BeEquivalent($"Betala {string.Format("{0:N2}", Convert.ToDecimal(products.Sum(x => x.UnitPrice / 100 * x.Quantity)))} kr")
-                    .Pay.ClickAndGo();
-            }
-        }
-
-
-        protected static IEnumerable TestData(bool singleProduct = true, string paymentMethod = PaymentMethods.Card)
-        {
-            var data = new List<object>();
-
-            if (singleProduct)
-                data.Add(new[]
-                {
-                    new Product { Name = Products.Product1, Quantity = 1 }
-                });
-            else
-                data.Add(new[]
-                {
-                    new Product { Name = Products.Product1, Quantity = 3 },
-                    new Product { Name = Products.Product2, Quantity = 2 }
-                });
-
-            switch (paymentMethod)
-            {
-                case PaymentMethods.Card:
-                    data.Add(new PayexCardInfo(TestDataService.CreditCardNumber, TestDataService.CreditCardExpiratioDate,
-                                               TestDataService.CreditCardCvc));
-                    break;
-
-                case PaymentMethods.Swish:
-                    data.Add(new PayexSwishInfo(TestDataService.SwishPhoneNumber));
-                    break;
-
-                case PaymentMethods.Invoice:
-                    data.Add(new PayexInvoiceInfo(TestDataService.PersonalNumberShort, TestDataService.Email, TestDataService.PhoneNumber,
-                                                  TestDataService.ZipCode));
-                    break;
-            }
-
-            yield return data.ToArray();
-        }
-    }
-}
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Atata;
+using Microsoft.Extensions.Configuration;
+using NUnit.Framework;
+using Sample.AspNetCore.SystemTests.PageObjectModels;
+using Sample.AspNetCore.SystemTests.PageObjectModels.Orders;
+using Sample.AspNetCore.SystemTests.PageObjectModels.Payment;
+using Sample.AspNetCore.SystemTests.PageObjectModels.ThankYou;
+using Sample.AspNetCore.SystemTests.Services;
+using Sample.AspNetCore.SystemTests.Test.Base;
+using Sample.AspNetCore.SystemTests.Test.Helpers;
+using SwedbankPay.Sdk;
+
+
+
+namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.Base
+{
+    public abstract class PaymentTests : TestBase
+    {
+        public PaymentTests()

```

Comment:
Codacy found an issue: [Change the visibility of this constructor to 'protected'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)